packages/blade/src/tokens/theme/createTheme.ts

```diff
@@ -0,0 +1,376 @@
+            600: brandColors[700],
+            700: brandColors[800],
+            800: brandColors[950],
+          },
+          gray: {
+            200: {
+              lowContrast: foregroundOnBrandColorPaymentLight,
+            },
+          },
+        },
+        action: {
+          background: {
+            primary: {
+              default: brandColors[600],
+              hover: brandColors[700],
+              focus: brandColors[800],
+              active: brandColors[900],
+            },
+            secondary: {
+              default: brandColors.a00,
+              hover: brandColors.a50,
+              focus: brandColors.a100,
+              active: brandColors.a200,
+            },
+            tertiary: {
+              default: globalColors.neutral.blueGrayLight[0],
+              hover: globalColors.neutral.blueGrayLight[50],
+              focus: globalColors.neutral.blueGrayLight[100],
+              active: globalColors.neutral.blueGrayLight[200],
+            },
+          },
+          border: {
+            primary: {
+              default: brandColors[600],
+              hover: brandColors[700],
+              focus: brandColors[800],
+              active: brandColors[900],
+            },
+            secondary: {
+              default: brandColors[600],
+              hover: brandColors[600],
+              focus: brandColors[600],
+              active: brandColors[600],
+            },
+            tertiary: {
+              default: globalColors.neutral.blueGrayLight[300],
+              hover: globalColors.neutral.blueGrayLight[300],
+              focus: globalColors.neutral.blueGrayLight[300],
+              active: globalColors.neutral.blueGrayLight[300],
+            },
+          },
+          text: {
+            primary: {
+              default: foregroundOnBrandColorPaymentLight,
+              hover: foregroundOnBrandColorPaymentLight,
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+          icon: {
+            primary: {
+              default: foregroundOnBrandColorPaymentLight,
+              hover: foregroundOnBrandColorPaymentLight,
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+        },
+      },
+    },
+  };
+
+  return overrideTheme({
```

Comment:
are you updating the documentation for override theme on storybook?

Reply:
Yes. In the followup PR